src/variables.json

```diff
@@ -60,9 +53,9 @@
   "white": "#fff",
   "transparent": "transparent",
 
-  "neutral10": "rgba(127, 127, 127, 0.1)",
-  "neutral25": "rgba(127, 127, 127, 0.25)",
-  "neutral45": "rgba(127, 127, 127, 0.45)",
+  "disabled-primary-interactive-color": "rgba(127, 127, 127, 0.25)",
+  "disabled-secondary-interactive-color": "rgba(127, 127, 127, 0.1)",
+  "disabled-text-color": "rgba(127, 127, 127, 0.45)",
```

Comment:
Ohh that is the one inaccurate use. I can change this to be 'placeholder-text-color'